### PR TITLE
[release-4.11.0] Bug 2097239: Changed Lower limits of CPU for PowerVS Cloud

### PR DESCRIPTION
**Pull request by bkhadars**
1. Changed Lower limit of CPU
2. Fixes the comment from the PR 5609: https://github.com/openshift/installer/pull/5609/files#r814418626

### CODE REVIEW

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: pkg/types/powervs/validation/machinepool.go [hunk 1]
```diff
@@ -43,18 +42,18 @@ func ValidateMachinePool(p *powervs.MachinePool, fldPath *field.Path) field.Erro
...
 	if p.Processors != "" {
-		processors, err := strconv.ParseFloat(p.Processors, 64)
+		_, err := strconv.ParseFloat(p.Processors, 64)
 		if err == nil {
-			if processors < 0.25 || processors > 32 {
-				allErrs = append(allErrs, field.Invalid(fldPath.Child("processors"), p.Processors, "number of processors must be from .25 to 32 cores"))
-			}
-			if math.Mod(processors*1000, 2) != 0 || math.Mod(processors*100, 25) != 0 {
-				allErrs = append(allErrs, field.Invalid(fldPath.Child("processors"), p.Processors, "processors must be in increments of .25"))
+			exp := `(\b^[0]?[.](5|50|75)$\b)|(\b^([1-9]|[12][0-9]|3[0-1])[.]?(25|5|50|75)?$\b)|(\b^32$\b)`
```
[the remark this answers is not in the document]
Comment (bkhadars) [reply pkg/types/powervs/validation/machinepool.go:48]: updated with coment

[the remark this answers is not in the document]
Comment (mkumatag) [reply pkg/types/powervs/validation/machinepool.go:48]: why we are restricting to 32 only? there are different possible numbers based on the system type you choose, for more information - https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-pricing-virtual-server

**Review by clnperez — CHANGES_REQUESTED** [review]

**Comment by clnperez** [comment]
@bkhadars can you add WIP to the title of this one until you've gotten the upper bounds checking done? and change the title to reflect its new purpose. thanks!

**Comment by clnperez** [comment]
n/m. it sounds like this PR will not be majorly reworked, and we'll just add the lower limts. still needs a rebase though @bkhadars

File: pkg/types/powervs/validation/machinepool.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (bkhadars) [reply pkg/types/powervs/validation/machinepool.go]: @mkumatag verifying upper limit will be reworked in the new PR

**Pull request merged**